examples/with-cloud/app/api/assistant-ui-token/route.ts

```
@@ -0,0 +1,55 @@
+  const userId = "usr_anon_" + generateId();
+  return userId;
+};
+
+const getJwtForUser = (userId: string) => {
+  return jwt.sign(
+    {
+      sub: userId,
+      iat: Date.now(),
+      exp: Date.now() + 60 * 60 * 24 * 7, // 1 week
+    },
+    process.env["JWT_SECRET"]!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)